src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,45 @@
+package tests;
+
+import org.testng.annotations.Test;
+import pages.HomePage;
+
+public class LoginTest extends BaseTest {
+    /*
+    * Ticket 2. Automate Login scenarios
+    * Login Empty credentials
+    * Login Wrong credentials (not existing email)
+    * Login Success.
+    *
+    * Note: Test sometimes will fail due to Capcha  Images
+    * */
+
+    @Test (priority = 0)
```

Comment:
default value of priority is 0. You can remove priority here and priority of this test will 0